partials/trial-banner.html

```diff
@@ -0,0 +1,5 @@
+<div id="trialBanner" class="ncf__trial-banner">
+		<p class="ncf__trial-banner-content">Your free {{#if trialDuration}}<span class="ncf__strong" >{{trialDuration}} </span>{{/if}}FT.com trial
```

Comment:
Just sorting out some spacing

```suggestion
		<p class="ncf__trial-banner-content">Your free {{#if trialDuration}}<span class="ncf__strong">{{trialDuration}}</span> {{/if}}FT.com trial
```

Reply:
this space was there for a reason. if we add `30 day` period, then there should be space after the word `day`.